src/components/PasswordStrength/PasswordStrength.module.scss

```diff
@@ -1,5 +1,43 @@
 @import '../../styles/mixins.scss';
 @import '../../styles/colors.scss';
+
+.Bubble {
+    width: 204px;
+    height: 53px;
+    border-radius: 3px;
+    box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.2);
+    background-color: #5c6f84;
```

Comment:
This colors is defined in colors.scss file, you can use "$color-blue-grey"

Reply:
ok use $color-blue-grey instead.